### PR TITLE
SC-5307: Import existing prerequisites and installation docs

### PR DESCRIPTION
**Pull request by andriitserkovnyi**
### Description

- Ticket/Issue: https://spryker.atlassian.net/browse/SC-5307

docker/sdk installation docs

#### Change log

import existing docs

### Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/06-installation/02-installation-guides/02-installation-guide-development-mode.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+```shell
+docker/sdk up
+```
+8. Update the `hosts` file:
+
+  - Linux/MacOS:				
+```shell
+echo "127.0.0.1 zed.de.spryker.local yves.de.spryker.local glue.de.spryker.local zed.at.spryker.local yves.at.spryker.local glue.at.spryker.local zed.us.spryker.local yves.us.spryker.local glue.us.spryker.local mail.spryker.local scheduler.spryker.local queue.spryker.local" | sudo tee -a /etc/hosts
+```
+@(Info)()(If needed, add corresponding entries for other stores. For example, if you are going to have a US store, add the following entries: `zed.us.spryker.local glue.us.spryker.local yves.us.demo-spryker`)
```
Comment (zyuzka) [inline docs/06-installation/02-installation-guides/02-installation-guide-development-mode.md:78]: same

**Review by zyuzka — CHANGES_REQUESTED** [review]

File: docs/06-installation/01-installation-prerequisites/02-linux-installation-prerequisites.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  "features" : {
+    ...
+    "buildkit" : true
+  }
+}
+```
+3. Restart Docker:
+```shell
+/etc/init.d/docker restart
+```
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/06-installation/01-installation-prerequisites/02-linux-installation-prerequisites.md:31]: Do you mean that the reader can optionally follow the instructions in the article you've linked?

File: docs/06-installation/02-installation-guides/01-modes-overview.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+* `user` - `spryker`
+* `pw` - `secret`
+
+You can change the credentials in the [Deploy file](https://documentation.spryker.com/docs/deploy-file-reference-10).
+
+### Debugging
+In Development mode, you can use [Xdebug](https://xdebug.org) for debugging.
+Run the command, to enable it:
+```bash
+docker/sdk {run|start|up} -x
```
[the remark this answers is not in the document]
Comment (andriitserkovnyi) [reply docs/06-installation/02-installation-guides/01-modes-overview.md:62]: Please suggest your changes directly :)

File: docs/06-installation/01-installation-prerequisites/02-linux-installation-prerequisites.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zyuzka) [reply docs/06-installation/01-installation-prerequisites/02-linux-installation-prerequisites.md:31]: By default, you can use docker with `sudo` only.
Those steps provide your user into the `docker` group

File: docs/06-installation/02-installation-guides/01-modes-overview.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (zyuzka) [reply docs/06-installation/02-installation-guides/01-modes-overview.md:62]:
```suggestion
docker/sdk {run|start|up|cli} -x
```

File: docs/06-installation/02-installation-guides/01-modes-overview.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+You can find file syncronization soulutions for each operating system in the table:
+
+| OS | Description |
+| --- | --- |
+| Linux | bind mount |
+| MacOS | [Mutagen](https://mutagen.io/) |
+| Windows | [docker-sync](http://docker-sync.io/) |
+
+Learn more about the solutions in respective documentation:
```
Comment (zyuzka) [inline docs/06-installation/02-installation-guides/01-modes-overview.md:35]: mutagen link

**Review by zyuzka — APPROVED** [review]

**Pull request merged**